src/main/java/com/nexmo/quickstart/verify/StartPsd2VerificationWithWorkflow.java

```diff
@@ -0,0 +1,33 @@
+package com.nexmo.quickstart.verify;
+
+import com.nexmo.client.NexmoClient;
+import com.nexmo.client.verify.Psd2Request;
+import com.nexmo.client.verify.VerifyResponse;
+import com.nexmo.client.verify.VerifyStatus;
+
+import static com.nexmo.quickstart.Util.configureLogging;
+import static com.nexmo.quickstart.Util.envVar;
+
+public class StartPsd2VerificationWithWorkflow {
+    public static void main(String[] args) {
+        configureLogging();
+
+        String NEXMO_API_KEY = envVar("NEXMO_API_KEY");
+        String NEXMO_API_SECRET = envVar("NEXMO_API_SECRET");
+        String RECIPIENT_NUMBER = envVar("RECIPIENT_NUMBER");
+        String PAYEE = envVar("PAYEE");
+        String AMOUNT = envVar("AMOUNT");
+
+
+
+        NexmoClient client = NexmoClient.builder().apiKey(NEXMO_API_KEY).apiSecret(NEXMO_API_SECRET).build();
+        VerifyResponse response = client.getVerifyClient()
```

Comment:
Should we consider adding our fancy new builder pattern here? Feel free to overrule me - but this would be the place to highlight it.

Reply:
1. Also, the VerifyClient acts as a wrapper around the actual request code so to make it so the users shouldn't have to use a builder to do basic request, the only time they would need to use the builder is if they are doing a more advanced request, also these snippets are following the code snippet specs. We didn't usually call out more advanced calls in our snippets other then what is in the specs but I can later add on that does a call with optional fields to show the builder or maybe I can just add it to the README.md
2. This is what I have for the jar when I do the _gradle build assemble_ command
![image](https://user-images.githubusercontent.com/13876064/92158071-5829de80-edf9-11ea-8b87-4617141039b3.png) if you changed the name of the project locally, that may change the name of the jar file I am assuming from the _acrhiveBaseName_ gradle property that is used in the build.gradle file

Reply:
1. Ok, so long as it's prominent in the Javadocs that works
2. Yeah, so I cloned the `vonage/vonage-java-code-snippets` rather than adding a second remote in the `nexmo/nexmo-java-code-snippets` repo - consequentially the project name is picked up as `vonage-java-code-snippets` - so I'm thinking that my point is valid - I would update the docs to represent what the default name of the JAR is going to be if this is pulled straight out of github.